packages/wix-storybook-utils/src/ui/typography.scss

```
@@ -76,6 +76,11 @@ $font: (
   }
 }
 
+/** This mixin is used to define the preview examples' font */
+@mixin previewExampleFont($type: madeFor) {
```

I think the argument can be removed
```suggestion
@mixin previewExampleFont() {
```